homeassistant/util/color.py

```diff
@@ -88,3 +87,11 @@ def color_xy_brightness_to_RGB(vX, vY, brightness):
     r, g, b = map(lambda x: int(x * 255), [r, g, b])
 
     return (r, g, b)
+
+
+def rgb_hex_to_list(hex_string):
```

Comment:
This will need some tests to make sure it works. I also think the name could be more descriptive, what about  `hex_to_rgb` ?